front/app/containers/Admin/settings/statuses/components/IdeationStatusCategories.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+
+import { NonLockedIdeationInputStatusCode } from 'api/idea_statuses/types';
+
+import RadioGroup from 'components/HookForm/RadioGroup';
+import Radio from 'components/HookForm/RadioGroup/Radio';
+
+import { MessageDescriptor, useIntl } from 'utils/cl-intl';
+
+import LabelText from './LabelText';
+import messages from './messages';
+
+const nonLockedIdeationInputStatusCodes: NonLockedIdeationInputStatusCode[] = [
+  'viewed',
+  'under_consideration',
+  'accepted',
+  'rejected',
+  'implemented',
+  'custom',
+];
+
+const ideationCodeTitleMessages: {
+  [key in NonLockedIdeationInputStatusCode]: MessageDescriptor;
+} = {
+  viewed: messages.viewedFieldCodeTitle,
+  under_consideration: messages.under_considerationFieldCodeTitle,
+  accepted: messages.acceptedFieldCodeTitle,
+  rejected: messages.rejectedFieldCodeTitle,
+  implemented: messages.implementedFieldCodeTitle,
+  custom: messages.customFieldCodeTitle,
+};
+const ideationCodeDescriptionMessages: {
+  [key in NonLockedIdeationInputStatusCode]: MessageDescriptor;
+} = {
+  viewed: messages.viewedFieldCodeDescription,
+  under_consideration: messages.under_considerationFieldCodeDescription,
+  accepted: messages.acceptedFieldCodeDescription,
+  rejected: messages.rejectedFieldCodeDescription,
+  implemented: messages.implementedFieldCodeDescription,
+  custom: messages.customFieldCodeDescription,
+};
```

Comment:
These will now show a warning if we add a new `NonLockedIdeationInputStatusCode`.